docs/api/public_php_api_managing_users.md

```diff
@@ -1,4 +1,4 @@
-# Managing Users [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
+# User API [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
 
```

Comment:
How about an introductory sentence? Now can be little confusing since the header is User and the next title is Segments.